src/client/menu/menu.c

```diff
@@ -76,13 +76,13 @@ qboolean m_entersound; /* play after drawing a frame, so caching won't disrupt t
 
 typedef struct
 {
-    void (*draw)(void);
```

Comment:
I know that the usage whitespaces instead of tabs in menu.c is annoying. Even more annoying is that I messed this up ~15 years when I reformated the code to a consistent style with `uncrustify`. But I think it's too late to change it to tabs, because it touches every line and breaks `git blame` and the like. With is very useful with the  error prone menu code and it's pointer magic. Therefor I would like to keep this file whitespace intended.

Reply:
I have removed code style changes commit, little code style change left only for functions where I have logic changes.

Reply:
It's possible to configure git to ignore certain commits
https://stackoverflow.com/questions/34957237/can-i-configure-git-blame-to-always-ignore-certain-commits-want-to-fix-git-blam